src/week4/FlagSuspiciousAdds.java

```diff
@@ -0,0 +1,19 @@
+package week4;
+
+import java.util.List;
+
+public class FlagSuspiciousAdds {
```

Comment:
Create Adds Interface and add method that flags out suspicious adds

Reply:
Okay hemant will do that